documentation/docs/40-best-practices/07-images.md

```diff
@@ -147,6 +147,8 @@ CDNs can generally be used without any need for a library. However, there are a
 - Consider serving all images via CDN regardless of the image optimization types you use. CDNs reduce latency by distributing copies of static assets globally.
 - Your original images should have a good quality/resolution and should have 2x the width it will be displayed at to serve HiDPI devices. Image processing can size images down to save bandwidth when serving smaller screens, but it would be a waste of bandwidth to invent pixels to size images up.
 - For images which are much larger than the width of a mobile device (roughly 400px), such as a hero image taking the width of the page design, specify `sizes` so that smaller images can be served on smaller devices.
-- Choose one image per page which is the most important/largest one and give it `priority` so it loads faster. This gives you better web vitals scores (largest contentful paint in particular).
-- Give the image a container or styling so that it is constrained and does not jump around. `width` and `height` help the browser reserving space while the image is still loading. `@sveltejs/enhanced-img` will add a `width` and `height` for you.
+- Choose one image per page that is above the fold and the most important (or largest) and give it `fetchpriority="high" decoding="sync" loading="eager"` so it loads as early as possible. This gives you better web vitals scores (largest contentful paint, in particular).
```

Comment:
I hadn't seen the `decoding="sync"` suggestion before. Any references you can share about that?
```suggestion
- Choose one image per page that is above the fold and the most important (or largest) and give it `fetchpriority="high" decoding="sync" loading="eager"` so it loads as early as possible. This gives you better web vitals scores (largest contentful paint (LCP), in particular).
```